Cargo.toml

```diff
@@ -82,7 +82,8 @@ rustc-std-workspace-std = { path = 'src/tools/rustc-std-workspace-std' }
 [patch."https://github.com/rust-lang/rust-clippy"]
 clippy_lints = { path = "src/tools/clippy/clippy_lints" }
 
-# When developing Yorick, you may find that you need to override ykpack.
-# You can do it once here, instead of all over the place.
-#[patch."https://github.com/softdevteam/yk"]
-#ykpack = { path = "../yk/ykpack" }
+# When developing Yorick, you may find that you need to override ykpack.  You
+# can do it once here, instead of all over the place, by uncommenting the two
+# following lines.
+[patch."https://github.com/softdevteam/yk"]
+ykpack = { git="https://github.com/ptersilie/yk", branch="projection" }
```

Comment:
I think this hack doesn't actually work. Did it work for you?

Reply:
Worked when I compiled manually. So would think it should work on bors too.

Reply:
The lock file refers to `ptersilie/yk`.

Reply:
We only need this to create and cache a build of rustc with which we can then merge changes in https://github.com/softdevteam/yk. Once we have a build, this is removed again before merging this PR.